Dangerfile.main.kts

```diff
@@ -24,4 +24,6 @@ if (danger.git.createdFiles.size + danger.git.modifiedFiles.size - danger.git.de
 
 if (danger.github!!.pullRequest.title.contains("WIP" ,false)) {
     warn("PR is classed as Work in Progress")
-}
\ No newline at end of file
+}
+
+warn("Test")
```

Comment:
You forgot to remove this one :)

Reply:
added with purpose.. where is it the danger comment??

Reply:
added with purpose. where is it the danger comment?

Reply:
https://github.com/danger/kotlin/pull/39#issuecomment-544179125